litmus-portal/graphql-server/manifests/cluster/1d_argo_deployment.yaml

```diff
@@ -79,10 +79,11 @@ spec:
             - workflow-controller-configmap
             - --executor-image
             -  #{ARGO-WORKFLOW-EXECUTOR}
+            - --container-runtime-executor #{ARGO-CONTAINER-RUNTIME-EXECUTOR}
```

Comment:
do you think it is better to stick to convention and add the placeholder in a subsequent ar list item? (similar to executor image etc.,)

Reply:
yes it would be better @ksatchit i am adding it that way.